Source/WebCore/html/PDFDocument.cpp

```diff
@@ -181,32 +181,48 @@ void PDFDocument::finishedParsing()
         sendPDFArrayBuffer();
 }
 
-void PDFDocument::sendPDFArrayBuffer()
+void PDFDocument::postMessageToIframe(const String& name, JSC::JSObject* data)
 {
-    using namespace JSC;
-
-    auto* frame = m_iframe->contentFrame();
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=236668 - Use postMessage
-    auto openFunction = frame->script().executeScriptIgnoringException("PDFJSContentScript.open"_s).getObject();
-
     auto globalObject = this->globalObject();
     auto& vm = globalObject->vm();
+    JSC::JSLockHolder lock(vm);
+
+    JSC::JSObject* message = constructEmptyObject(globalObject);
+    message->putDirect(vm, JSC::Identifier::fromString(vm, "message"_s), JSC::jsNontrivialString(vm, name));
```

Comment:
nit: You can use the "message" identifier that's already baked inside CommonIdentifiers.